tests/access_tests.py

```diff
@@ -428,6 +428,14 @@ def test_approve(self, mock_send_mime):
             session.delete(security_manager.find_role(TEST_ROLE_NAME))
             session.commit()
 
+    def test_dashboard_endpoint_malicious_redirect(self):
+        # todo(gianluca.ciccarelli@bolt.eu): I need a dashboard ID that will
```

Comment:
You could try using `-1` (though I’m unsure if that works), otherwise maybe querying for the maximum dashboard ID and adding one, or using `9999`.